apis/druid/v1alpha1/druid_types.go

```diff
@@ -91,9 +91,16 @@ type DruidSpec struct {
 	// +optional
 	DeleteOrphanPvc bool `json:"deleteOrphanPvc"`
 
-	// Required: path to druid start script to be run on container start
+	// Required: Command to be run on container start
 	StartScript string `json:"startScript"`
 
+	// Optional: bash/sh entry arg. Set startScript to `sh` or `bash` to customize entryArg
+	// For example, the container can run `sh -c "${EntryArg} && ${DruidScript} {nodeType}"`
```

Comment:
I don't think this is fully backwards compatible. Since we are now wrapping this in a shell, any signals sent to the process running the container will be sent to the shell and not to the JVM. The druid startup script ensures that the JVM will replace the shell, but in this case I do not think that will happen.

Is there a reason why the changes cannot be done inside of the druid startup script? It might be useful to provide an example use-case to clarify that.

Reply:
It just gives the user an option to run druid startup script in shell, but the default is still 
https://github.com/druid-io/druid-operator/pull/268/files#diff-6b7cea906298d4262b867f6e81f41caa89c2daeb67247bd95169e93ed80a00b4R1146

Reply:
understood, but having it behave differently is not a good idea, and the nuances of using one or the other are too subtle in my opinion.

Reply:
Its true the same can be achieved by adapting `druid.sh`. Adding custom arguments just provides a more lightweight way to update the container startup command than updating `druid.sh` and building a new image.

Reply:
understood, but can we include a concrete use-case in the description that would warrant this feature specifically in operator? it would help understand the motivation for this change.

Reply:
wouldn't it make more sense to let someone override the entry point in the helm chart rather than having to customize it in the operator config?